graphics/cairo-devel/Portfile

```diff
@@ -51,12 +52,6 @@ depends_lib                 path:lib/pkgconfig/pixman-1.pc:libpixman \
 
 minimum_xcodeversions       {8 2.4.1}
 
-# Prevent cairo from using librsvg, libspectre, poppler.
-patchfiles-append           patch-configure.diff
```

Comment:
You've removed the Portfile code here that ensured that cairo would not use librsvg, libspectre, or poppler, and below for ghostscript or lzo2, even if they were installed. What's the status of each of these now? Have you verified that, if those are installed, they aren't used? (Does the meson script disable these features by default and require you to opt into them?)

Reply:
I didn't validate that this is covered. It needs more work.

Reply:
BTW: I don't see where I've removed the ghostscript or lzo2 checks.

Reply:
The key output I see during the configure phase -- without trace mode, but with the various dependencies installed -- is:

```
Requested 'libspectre >= 99999' but version of libspectre is 0.2.12
configure:25672: WARNING: PDF backend will not be tested since poppler >= 99999 is not available
configure:26038: WARNING: SVG backend will not be tested since librsvg >= 99999 is not available
```

And the configure script defines the following variables, which appear to be utilized/imported by the Meson build (?), are defined as (matching the previous output):

```
configure:SPECTRE_VERSION_REQUIRED=99999
configure:POPPLER_VERSION_REQUIRED=99999
configure:LIBRSVG_VERSION_REQUIRED=99999
```

I haven't had a chance to review upstream's commit history yet. But is the current release setup to explicitly avoid these three dependencies? (Superficially that appears to be the case, but curious whether you folks have more insight into the details...)

Reply:
Sorry for the confusion, my previous message (RE: various versions required for dependencies) was due to working with the wrong branch (1.17.6, rather than Lukas' 1.17.8).

Never Mind...